### DISCUSSION TITLE
Graphical project manager for linux

### DISCUSSION THREAD

**Discussion by tct123** [o3de/o3de#17836] [Ideas]
Hello, i think it would be nice to have a gui project manager for linux as well. just for simplicity.

**Comment by Andre-LA** [comment]
Running the `o3de` executable does open the the project manager for me (Fedora Linux 39), I did compiled from source though, how did you installed o3de?

![o3de project manager](https://github.com/o3de/o3de/assets/8538122/c560e52c-4c35-4e20-a572-cd4d85616938)
